tests/test_data_validation_before_insert.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+import pandas as pd
+import logging
+from shared_code.utils import validate_data
+
+logger = logging.getLogger(__name__)
+
+_i = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+if _i not in sys.path:
+    # add parent directory to sys.path so utils module is accessible
+    sys.path.insert(0, _i)
+del _i  # clean up global name space
+
+
+# This scrapper tests if the script validator can remove any rows with invalid data
+class TestDataScraper():
```

Comment:
No need to have a class TestDataScraper() , you should test the original class instead. In this case since you have moved the method to utils.py , you can just test the validate_data method